cmd/issue_edit.go

```diff
@@ -26,7 +26,8 @@ var issueEditCmd = &cobra.Command{
 		lab issue edit 14 -m "new title" -m "new desc"
 		lab issue edit 14 -l new_label --unlabel old_label
 		lab issue edit --milestone "NewYear"
-		lab issue edit --force-linebreak`),
+		lab issue edit --force-linebreak
+		lab issue edit 14 --deletenote <noteID>`),
```

Comment:
I have the impression that this help message is completely wrong 😄 .
It should be `lab issue edit [remote] <mr_id>:<comment_id>`. the example leads to an `invalid remote` sort of error.
And the same happens in the mr_edit.go code.

Reply:
Oh of course.  That has to be 'lab issue edit <id>:<comment_id>'.  I've fixed it in both cases.